c7n/resources/fsx.py

```diff
@@ -16,6 +24,7 @@ class resource_type(TypeInfo):
         service = 'fsx'
         enum_spec = ('describe_file_systems', 'FileSystems', None)
         name = id = 'FileSystemId'
+        filter_name = 'MaxItems'
```

Comment:
that's not really the filter name here, thats for when we can pass through on ids, in this case its a static param.

Reply:
Updated to use the `FileSystemIds`